client/src/containers/Sensor/components/SensorTable.tsx

```diff
@@ -1,22 +1,46 @@
-import React, { useEffect } from 'react'
-import { Sensor } from '../../../model/sensor'
-import { Table, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core'
+import React, { useEffect, useState, useCallback } from 'react'
+import { apiResponse } from '../../../epics/utils'
+import { Button, Table, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core'
+import { Sensor, SensorCreateParams } from '../../../model/sensor'
+import SensorCreation from './SensorCreation'
 
 interface Props {
-  onGetSensorList: () => void
+  onCreateSensor: (sensorBase: SensorCreateParams) => void
+  onGetSensorList: (skip: number, limit: number) => void
   sensorList: Sensor[]
 }
 
 export default function SensorTable(props: Props) {
-  const { onGetSensorList, sensorList } = props
+  const { onCreateSensor, onGetSensorList, sensorList } = props
+  const [showForm, toggleForm] = useState(false)
```

Comment:
```suggestion
  const [isFormVisible, setFormVisible] = useState(false)
```
`showForm` is verb, so I will expect it to be a function.
`toggle` means turning state from `false` to `true` or  from `true` to `false`, I will expect it accepts no parameters.